src/commands/convert/hdf2caat.ts

```diff
@@ -0,0 +1,27 @@
+import {Command, Flags} from '@oclif/core'
+import fs from 'fs'
+import path from 'path'
+import {FromHDFToCAATMapper as Mapper} from '@mitre/hdf-converters'
+
+export default class HDF2CAAT extends Command {
+  static usage = 'convert hdf2caat -i <hdf-scan-results-json>... -o <output-caat-xlsx> [-h]'
+
+  static description = 'Translate an HDF file into a CAAT XLSX'
```

Comment:
Can you say what CAAT is here?

Reply:
that was a request from eugene as well.  will be addressed